api/src/main/java/com/valtech/aem/saas/api/query/FilterJoinOperator.java

```diff
@@ -0,0 +1,15 @@
+package com.valtech.aem.saas.api.query;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+public enum FilterJoinOperator {
+  AND(" "),
```

Comment:
shouldn't this be " AND " as well?

Reply:
I assumed a blank space is considered an "AND". But i guess I should specify it as " AND ".

Reply:
not 100% sure, can you doublecheck please?

Reply:
I checked it. blankspace is considered " OR ".